src/containers/App.js

```diff
@@ -187,9 +187,24 @@ function App() {
     });
   }
 
+  function addCard() {
+    const cardNew = {
+      id: cardState.cards[cardState.cards.length-1].id + 1,
```

Comment:
Better generate it via uuid package.
Otherwise, if you have your id as a string - it will not increment but will concatenate.